interface.go

```diff
@@ -325,6 +324,10 @@ type Config struct {
 	// This can be useful if version information is exchanged out-of-band.
 	// It has no effect for a client.
 	DisableVersionNegotiationPackets bool
+	// Allow0RTT allows the application to decide if a 0-RTT connection attempt should be accepted.
+	// When set, 0-RTT is enabled. When not set, 0-RTT is disabled.
+	// Only valid for the server.
+	Allow0RTT func(net.Addr) bool
```

Comment:
This makes sense, but what's the use case for custom logic to switch depending on address?

Reply:
The address is not particularly useful indeed, especially since it's not validated yet at this point.

On the other hand, I really want to introduce _some kind of_ callback here, because it might be interesting to have the application make the decision based on _something_. For example, we could allow the application to save some state into the session ticket at some point, and then allow / reject 0-RTT based on the restored stayed when the connection is resumed.

I'll add a comment saying that this API is unstable and we'll add more fields to the callback at some point.